local-running/src/main/java/uk/gov/di/ipv/coreback/handlers/LambdaHandler.java

```diff
@@ -260,7 +261,7 @@ private CriCallbackRequest buildCriCallbackRequest(Request request) {
                 .error(requestBody.get("error"))
                 .errorDescription(requestBody.get("errorDescription"))
                 .ipAddress(request.headers(IP_ADDRESS))
-                .featureSet(request.headers(FEATURE_SET))
+                .featureSet(Arrays.asList(request.headers(FEATURE_SET).split(",")))
```

Comment:
This is using `Arrays.asList` but the others are using `Stream.of(..).toList()`?

This seems cleaner (and avoids an unnecessary stream), so maybe prefer that? If you create a helper for this then you can make sure it's uniform everywhere :)

Reply:
Done